src/mlpack/methods/decision_tree/decision_tree_impl.hpp

```diff
@@ -13,6 +13,7 @@
 #define MLPACK_METHODS_DECISION_TREE_DECISION_TREE_IMPL_HPP
 
 #include "decision_tree.hpp"
+#include <mlpack/core.hpp>
```

Comment:
Shouldn't this already be included by `decision_tree.hpp`?  If `decision_tree.hpp` actually only includes `mlpack/prereqs.hpp`, I think it would be fine to add `size_checks.hpp` to `prereqs.hpp` instead of `core.hpp`. :+1:

Reply:
Okay then, moving to `prereqs.hpp` :)